modules/display.c

```diff
@@ -6935,25 +6998,16 @@ static gboolean mdy_dbus_send_blanking_inhibit_status(DBusMessage *const method_
      */
     if( display_state == MCE_DISPLAY_ON ||
         display_state == MCE_DISPLAY_DIM ) {
-            curr = !(mdy_blanking_off_cb_id || mdy_blanking_dim_cb_id);
-
+        if( !mdy_blanking_off_cb_id && !mdy_blanking_dim_cb_id )
+            curr = true;
```

Comment:
Not really part of the diff, but because it touches it: Maybe `curr` can be renamed to `blanking_active` or something more descriptive?